README.md

```diff
@@ -1,6 +1,21 @@
 # Planetary
 
-Planetary iOS is an app based on [Secure Scuttlebutt](https://scuttlebutt.nz/) that doesn't keep your data in the cloud and allows you and your friends to come together and connect even when the internet goes out.  
+Social media for humans, not algorithms.
+
+Planetary passes data from friend to friend, creating a network that is personal, censorship resistant, and minimizes abuse and spam. Planetary gives you ownership over your relationships and content, and is compatible with any app using the Secure Scuttlebutt protocol.
+
+In Planetary there are no advertisements or artificial intelligence algorithms trying to make you feel a certain way. Your content lives on your device and the devices of those around you, it isn't owned or controlled by a corporation. Experience social networking where you're the customer, not the product.
+
+Unlike most social media, Planetary:
+* Does not sell your data
+* Is naturally spam-resistant
+* Compatible with other apps including Manyverse, Patchwork, and more
+* Works off-grid and offline.
+* Allows you to take your friends with you if you decide to move to another client.
+* Is open source!
+
+No one company should own the Internet’s public spaces, which is why we’re building on —and contributing back to— the open source Secure Scuttlebutt (SSB) ecosystem. Their core technologies let us recreate a social network experience, but in a decentralized way that no one organization can dominate.
```

Comment:
```suggestion
No one company should own the Internet’s public spaces, which is why we’re building on —and contributing back to— the open source [Secure Scuttlebutt](https://scuttlebutt.nz/) ecosystem. Their core technologies let us recreate a social network experience, but in a decentralized way that no one organization can dominate.
```

Reply:
Should we make this a link so that it is easier to read about it?

Reply:
yes!